services/app-web/src/contexts/TraceContext.tsx

```diff
@@ -1,9 +1,7 @@
 import React from 'react'
 import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
 import { WebTracerProvider } from '@opentelemetry/sdk-trace-web'
-import { BaseOpenTelemetryComponent } from '@opentelemetry/plugin-react-load'
-import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http'
-import { diag, DiagConsoleLogger } from '@opentelemetry/api'
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http/build/esnext'
```

Comment:
This one weird importing path took a while to figure out. There is an error that occurs in the `0.28` version of the `exporter-trace-otlp-http` library that is fixed in `0.29.2`. However, going to `0.29` line of packages isn't an option for us with our current setup. Importing direct from `esnext` fixes it in this line.

Reply:
urg. what a world.